users/api/clients.go

```diff
@@ -164,7 +171,7 @@ func clientsHandler(svc users.Service, authn mgauthn.Authentication, tokenClient
 		// SpiceDB provides list of user ids in given user_group_id
 		// and users service can access spiceDB and get the user list with user_group_id.
 		// Request to get list of users present in the user_group_id {groupID}
-		r.Get("/groups/{groupID}/users", otelhttp.NewHandler(kithttp.NewServer(
+		r.Get("/domains/{domainID}/groups/{groupID}/users", otelhttp.NewHandler(kithttp.NewServer(
```

Comment:
There is no need for `/domains` prefix here, use `/{domainID}/groups/...`. I.e. unlike other entities, let's not prefix domain ID.

Reply:
Same for other endpoints.